apps/site/assets/css/_events.scss

```diff
@@ -70,8 +70,13 @@ $mobile-control-height: 73px;
 
   a {
     display: block;
-    font-weight: bold;
+    font-size: 18px;
+    font-weight: 400;
     margin-bottom: $base-spacing / 2;
+
+    &.m-primary-month {
+      font-weight: $headings-font-weight;
```

Comment:
What if in the future we decide headings should have a lighter weight? I think it makes more sense to just do `bold` here.